src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion_adapter.py

```diff
@@ -0,0 +1,799 @@
+# Copyright 2022 The HuggingFace Team. All rights reserved.
```

Comment:
Can we move the whole pipeline into a new folder / file here ? The SD folder is getting a bit too full. Maybe `src/diffusers/pipelines/t2i_adapter/...`

Reply:
done!